app/src/main/java/com/example/firstapplication/MainActivity.java

```diff
@@ -35,6 +35,8 @@ public void onClick(View v) {
                 viewPassword.setText( editPassword.getText().toString());
```

Comment:
Use the same naming scheme for textview variables here.

When naming buttons start with btn so this button will be btnRegister.